src/lib/esnext.globalthis.d.ts

```diff
@@ -0,0 +1 @@
+declare var globalThis: typeof globalThis;
```

Comment:
Is this lib needed? Doesn't the default lookup behavior just make this available in every target?

Reply:
Yes! This is left over from my previous approach.